index.js

```diff
@@ -56,5 +58,11 @@ class BootSimulator {
   }
 }
 
+let myArgs = require('optimist').argv;
```

Comment:
instead of myArgs can we have something meaningful

Reply:
Will change it to simBootParams?